src/components/register.js

```diff
@@ -82,7 +82,7 @@ export default class RegisterComponent extends React.Component {
           <p>Connect with parents and education professionals from around the world to make education better for all children in all schools and families worldwide.</p>
         </div>
       </header>
-      <form action="" onSubmit={this.submitHandler} className="layout__row">
+      <form action="" onSubmit={this.submitHandler} onClick={this.props.onClick} className="layout__row">
```

Comment:
Moved click handler to div wrapper that contains `RegisterComponent` and is placed into `Auth.render()`. Now any click at `RegisterComponent` (that is not only form) calls the handler.